axum-core/src/extract/from_ref.rs

```diff
@@ -5,10 +5,9 @@
 ///
 /// See [`State`] for more details on how library authors should use this trait.
 ///
-/// This trait can be derived using `#[derive(axum_macros::FromRef)]`.
+/// This trait can be derived using `#[derive(FromRef)]`.
```

Comment:
The link on this didn't work for some reason but I don't actually think we need it since the macro is exported at the same path as `FromRef` itself.

Reply:
Well, there were no square brackets around the code block.

Reply:
Even with I couldn't get it working 🤷